src/main/webapp/WEB-INF/view/navigationbar.jsp

```diff
@@ -1,19 +1,19 @@
 <!DOCTYPE html>
    <link rel="stylesheet" href="/css/navigation.css">
   <nav>
-    <a id="navTitle" href="/">CodeU Chat App</a>
-    <a href="/conversations">Conversations</a>
+    <a id="navTitle" href="/">ChatU</a>
+    <a href="/conversations" id="nav">Conversations</a>
```

Comment:
Since there are multiple navigation links that you want to style the same way, I think the best practice is to use a class instead of an ID: https://stackoverflow.com/a/12889421

Reply:
Ohhhh that would definitely make more sense